### PR TITLE
[10.0][MIG] sale_fixed_discount

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration based on https://github.com/OCA/sale-workflow/pull/522.

* [x] : Depends on https://github.com/OCA/account-invoicing/pull/353

Sale Fixed Discount
===================

This module extends the functionality of Sales to allow you to apply fixed
amount discount at sales order line level.

The module also extends the sales order report to show fixed discount.

Configuration
=============

To configure this module, you need to:

#. Go to *Sale > Configuration >Settings*.
#. In the *Discount* option select *Allow discounts on sales order lines*.

Usage
=====

To use this module, you need to:

#. Go to *Sales*.
#. Create a Sales Order and specify the type of discount and fixed/percent
   discount in a line.

### CODE REVIEW

File: sale_fixed_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (MiquelRForgeFlow) [inline sale_fixed_discount/README.rst]: png

File: sale_fixed_discount/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (MiquelRForgeFlow) [inline sale_fixed_discount/README.rst]: Icon https://odoo-community.org/logo.png`_.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: sale_fixed_discount/README.rst (hunk 2 above)
Comment (LoisRForgeFlow) [reply sale_fixed_discount/README.rst]: done

File: sale_fixed_discount/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Fixed Discount",
+    "summary": "Allows to apply fixed amount discounts in sales orders.",
+    "version": "10.0.1.0.0",
+    "category": "Sales",
+    "website": "https://odoo-community.org/",
```
Comment (JayVora-SerpentCS) [inline sale_fixed_discount/__manifest__.py]: https://github.com/OCA/sale-workflow

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: sale_fixed_discount/__manifest__.py (hunk 3 above)
Comment (LoisRForgeFlow) [reply sale_fixed_discount/__manifest__.py]: done.

**Comment by LoisRForgeFlow** [comment]
@JayVora-SerpentCS Could you please update your review?

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Ready to merge I think.

**Pull request merged**